HPXMLtoOpenStudio/measure.rb

```diff
@@ -4192,6 +4289,38 @@ def self.get_foundation_and_walls_top()
     walls_top = foundation_top + 8.0 * @ncfl_ag
     return foundation_top, walls_top
   end
+  
+  def self.get_location_hierarchy(ba_cz_name)
+    if [Constants.BAZoneHotDry, Constants.BAZoneHotHumid].include? ba_cz_name
+      return [HPXML::LocationGarage,
+              HPXML::LocationLivingSpace,
+              HPXML::LocationBasementConditioned,
+              HPXML::LocationInterior,
```

Comment:
You'll be removing this, but note that this choice didn't make any sense. If you look at EPvalidator, you'll see that this isn't a valid water heater location.